### DISCUSSION TITLE
Finding the reason for numerical instability in FDS

### DISCUSSION THREAD

**Discussion by vijay00901** [firemodels/fds#13725] [Q&A]
Dear friends, 

I am running a simiulation in FDS and it is causing numerical instability. I am trying to figure out the reason for that. Would someone be able to point me in the right direction. I cannot attach certain files so I am sharing it here. Thank you so much.
![image](https://github.com/user-attachments/assets/71347bc7-7953-4c7c-818a-146d2fdc05a9)
https://drive.google.com/drive/folders/1eA7058j7EIW48ynioIjA2vIaxa2t80P9?usp=drive_link

**Comment by vijay00901** [comment]
Hi friends, I found out that the initial time step was 0.0583 and the time step at which instability occurred was 6.13 E-6. 
There is a condition i.e. if the time step goes below 0.0001 that of initial time step, calculation stops. The value I got is very close to this limit. should I change the LIMITING_DT_RATIO? Please advise.

**Comment by rmcdermo** [comment]
I cannot look at the Google Drive you posted.  Please add the file extension ".txt" to your input file and post it here.

**Reply by vijay00901** [reply]
Thank you Dr. McDermott. I have attached the files here. 
[error_file.txt](https://github.com/user-attachments/files/17684306/error_file.txt)
[out_file.txt](https://github.com/user-attachments/files/17684307/out_file.txt)
[model3_0_1g_steps.csv](https://github.com/user-attachments/files/17684310/model3_0_1g_steps.csv)

**Reply by rmcdermo** [reply]
You have not included an input file.  From the .out file, I cannot really tell what is wrong.  Also, please try with the latest code.  Quite possible we have sorted out a problem that will help.

**Comment by vijay00901** [comment]
Sorry I forgot to include that. I have attached it here
[input_file.txt](https://github.com/user-attachments/files/17684580/input_file.txt)

**Comment by rmcdermo** [comment]
OK, so we are doing microgravity combustion with a premixed flame?

Please let's start simple.  You are way into research territory here and the very first thing you send me is super complicated.  Can you just do a simple flame speed calculation with the chemistry you are giving FDS?  Try that with the latest code and report back.

**Reply by vijay00901** [reply]
Yes Dr. McDermott. I am actually doing research. What you said will be a digression from my end goal. I got the simulation run successfully for various gravity values like 0.5g, 0.3g, 0.2g but 0.1g is having this issue. Is there any way I can pinpoint the issue or solve it? Thank you so much.

**Reply by mcgratta** [reply]
If you are doing research and you do not do the simple verification exercise that @rmcdermo suggests, then your research is doomed to failure. You are using FDS in a way that most people do not normally use it. It is very important that you check to ensure that the model assumptions are appropriate for your case.
